autoPyTorch/pipeline/components/setup/network/base_network.py

```diff
@@ -26,6 +26,7 @@ def __init__(
     ) -> None:
         super(NetworkComponent, self).__init__()
         self.network = network
+        self.network_snapshots: List[torch.nn.Module] = []
```

Comment:
Can you add this to the constructor? The thing with scikit learn is that it can use joblib to launch the job in a different process, so when it rebuilds the fitted object it will lose this snapshot attribute, which will be troublesome for debug.

Reply:
okay makes sense